packages/divi-scripts/template/includes/fields/Input/Input.jsx

```diff
@@ -0,0 +1,40 @@
+// External Dependencies
+import React, { Component } from 'react';
+
+// Internal Dependencies
+import './style.css';
+
+class Input extends Component {
+
+  static slug = '__prefix_input';
+
+  constructor(props) {
+    super(props);
+    this._onChange = this._onChange.bind(this);
```

Comment:
why not using arrow function? iirc we can remove the manual bind if `_onChange` is declared using arrow function

Reply:
@fikrirasyid Thanks! I just know it now 😬 Addressed your feedback.